compose/install.md

```diff
@@ -95,185 +92,120 @@ on Microsoft Windows Server and want to install Docker Compose.
+2. Apply executable permissions to the binary:
 
-#### Install as a container
+  ```console
+  $ sudo chmod +x /usr/local/bin/docker-compose
+  ```
+  
+  > **Note**:
+  >
+  > If the command `docker-compose` fails after installation, check your path.
+  > You can also create a symbolic link to `/usr/bin` or any other directory in your path.
+  >
+  > For example:
+  > ```console
+  > $ sudo ln -s /usr/local/bin/docker-compose /usr/bin/docker-compose
+  > ```
 
-Compose can also be run inside a container, from a small bash script wrapper. To
-install compose as a container run this command:
-
-```console
-$ sudo curl -L --fail https://github.com/docker/compose/releases/download/{{site.compose_version}}/run.sh -o /usr/local/bin/docker-compose
-$ sudo chmod +x /usr/local/bin/docker-compose
-```
-
-</div>
-
-<div id="pre-release" class="tab-pane fade" markdown="1">
-
-### Install pre-release builds
-
-If you're interested in trying out a pre-release build, you can download release
-candidates from the [Compose repository release page on GitHub](https://github.com/docker/compose/releases){: target="_blank" rel="noopener" class="_"}.
-Follow the instructions from the link, which involves running the `curl` command
-in your terminal to download the binaries.
-
-Pre-releases built from the "master" branch are also available for download at
-[https://dl.bintray.com/docker-compose/master/](https://dl.bintray.com/docker-compose/master/){: target="_blank" rel="noopener" class="_"}.
-
-> Pre-release builds allow you to try out new features before they are released,
-> but may be less stable.
-{: .important}
+3. Test the installation.
 
+    ```console
+    $ docker-compose --version
+    Docker Compose version {{site.compose_version}}
+    ```
 </div>
 </div>
 
 ----
 
-## Upgrading
-
-If you're upgrading from Compose 1.2 or earlier, remove or
-migrate your existing containers after upgrading Compose. This is because, as of
-version 1.3, Compose uses Docker labels to keep track of containers, and your
-containers need to be recreated to add the labels.
-
-If Compose detects containers that were created without labels, it refuses
-to run, so that you don't end up with two sets of them. If you want to keep using
-your existing containers (for example, because they have data volumes you want
-to preserve), you can use Compose 1.5.x to migrate them with the following
-command:
-
-```console
-$ docker-compose migrate-to-labels
-```
-
-Alternatively, if you're not worried about keeping them, you can remove them.
-Compose just creates new ones.
-
-```console
-$ docker container rm -f -v myapp_web_1 myapp_db_1 ...
-```
-
 ## Uninstallation
 
 To uninstall Docker Compose if you installed using `curl`:
 
-```console
-$ sudo rm /usr/local/bin/docker-compose
-```
-
-To uninstall Docker Compose if you installed using `pip`:
-
-```console
-$ pip uninstall docker-compose
-```
+  ```console
+    $ rm $DOCKER_CONFIG/cli-plugins/docker-compose
+  ```
```

Comment:
Indentation on these looks "off"; when using code-fences, they should be without indentation (and no indentation within the block itself)